web3.js/test/connection.test.ts

```diff
@@ -2002,6 +2003,32 @@ describe('Connection', () => {
       await connection.removeRootChangeListener(subscriptionId);
     });
 
+    it('logs notification', async () => {
+      let listener: number | undefined;
+      const owner = new Account();
+      const [logsRes, ctx] = await new Promise(resolve => {
+        const filter = 'all' as LogsFilter;
```

Comment:
This is awkward, but I'm not sure how to make TypeScript happy without the cast.

Reply:
Haha, TypeScript was trying to tell you that your logs filter typedef has an issue: `'all '` (note the extra space)